### DISCUSSION TITLE
Demonstration will be helpful with csv data file

### DISCUSSION THREAD

**Discussion by tuhinPhd** [community/community#123197] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear Professor, 
It will be helpful if i just get a working sample with real csv data.I am stuck.Could you please help me,?

Thanks

Tuhin

**Comment by MostlyKIGuess** [comment]
Try using panda maybe?

**Comment by tuhinPhd** [comment]
Yes Professor,
I used pd
And I could not assign value to the varriables.

By the way,i also want to know is there any way to take training courses?

Tuhin Al Mamun

**Comment by ebndev** [comment]
Hello! Thank you for being a part of the GitHub Community!

Please note: GitHub Discussions is a public forum and not intended to be used as a private message board. As a result, I will be locking this post but please feel encouraged to post again in the future if you have questions or topics you'd like to share with others.

Thanks for understanding!
